content/en/docs/ops/deployment/vm-architecture/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Virtual Machine Architecture
+description: Describes Istio's high-level architecture for virtual machines.
+weight: 20
+keywords:
+- virtual-machine
+test: n/a
+owner: istio/wg-environments-maintainers
+---
+
+Before reading this document, be sure to review [Istio's architecture](/docs/ops/deployment/architecture/) and [deployment models](/docs/ops/deployment/deployment-models/).
+This page builds on these pages to explain how Istio can be extended to support joining virtual machines into the mesh.
+
+Istio's virtual machine support involves connecting workloads outside of a Kubernetes cluster to the mesh.
+This enables legacy applications, or applications not suitable to run inside of Kubernetes, to get all the same benefits Istio provides to applications running inside Istio.
+
+Many of the features the Kubernetes automatically provides applications are supplemented by Istio.
+This includes service discovery, DNS resolution, and health checks.
+Additionally, standard Istio features such as automatic mutual TLS, rich telemetry, and expressive traffic management configuration are enabled.
+
+The following diagram shows the architecture of a mesh with virtual machines:
+
+{{< tabset category-name="network-mode" >}}
+
+{{< tab name="Single-Network" category-value="single" >}}
+
+In this mesh, there is a single [network](/docs/ops/deployment/deployment-models/#network-models), meaning pods and virtual machines can directly reach each other.
+
+Control plane traffic, including XDS configuration and certificate signing, are sent through a Gateway in the cluster.
```

Comment:
Is this an ingress/egress GW? Does this matter which one?

Reply:
It can be any gateway with istiod exposed. It doesn't even need to be an istio gateway technically. Our docs recommend a dedicated gateway.